subprojects/docs/src/samples/testing/jacoco/quickstart/build.gradle

```diff
@@ -56,3 +56,27 @@ jacocoTestReport {
     }
 }
 // END SNIPPET report-configuration
+
+// START SNIPPET violation-rules-configuration
+jacocoTestCheck {
+    violationRules {
+        rule {
+            limit {
+                minimum = 0.5
```

Comment:
Nitpick for the JaCoCo maintainers: "INSTRUCTION" is a weird default. Users are going to want to set this to something. Copy and pasting this example is kinda a weird experience (at least it was for me) because INSTRUCTION coverage is 0 on my projects.